src/telegram/process_update_task.rs

```diff
@@ -28,6 +28,15 @@ use std::fmt::Write;
 
 const BOT_NAME: &str = "RustWeather77Bot";
 
+pub struct Params<'a> {
```

Comment:
is there any reason to store references? Why not own the values?

Reply:
[Params without refs](https://github.com/pxp9/weather_bot_rust/pull/12/commits/6c58a0147c267ad1cf98985b4fcabc7dc5f8bd60)